packages/devtools-launchpad/src/index.js

```diff
@@ -93,19 +93,7 @@ function getTargetFromQuery() {
   return null;
 }
 
-function bootstrap(React, ReactDOM, App, appActions, appStore) {
-  const connTarget = getTargetFromQuery();
-  if (connTarget) {
```

Comment:
we should update this function so that it is *just* getting tabs. 

currently we're also bootstrapping here, which is silly. We only want to establish the connection once